code-challenges.test.js

```diff
@@ -22,11 +40,47 @@ const fibLength2 = 10
 
 // b) Create the function that makes the test pass.
 
+//Psuedocode 
+// make a function named fibanacci maker.
+// declare an array with 1,1 in it to use for the first two spots
+// use a loop with the index starting at two make the lenght the variable number
+// have the loop go forwards by using i++
+// make a newFibanacci variable to make the new sequence
+// concat [i-1]+[i-2]
+// use the push property to return the first fibanacci with the second fibbanacci 
+
+const fib = (num) => {
+  let fibonacci = [1, 1];
+  for(let i = 2; i < num; i++){
+    let newFib = fibonacci[i-1] + fibonacci[i-2]
+    fibonacci.push(newFib)
+  } 
+  return fibonacci
+}
+
+// the test did not pass 
+// // Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        1.103 s
+// Ran all test suites.
+
+// im not sure why the test isnt passing everything seems right to me. 
```

Comment:
Your code looks great! I would encourage you to look at your test, theres a couple of different function names.